app/i18n/cz/user.php

```diff
@@ -38,4 +38,7 @@
 			'invalid' => 'You must accept the Terms of Service to be able to register.',	// TODO - Translation
 		),
 	),
+	'username' => array(
+		'invalid' => 'The username is invalid.',	// TODO - Translation
```

Comment:
I've noticed what's presumably a security practice of not visibly differentiating between invalid user & password. (As a user I'd find it annoying; just putting it out there for the moment.)

Reply:
This message is used during registration, not during login :) The message during login is actually "Login is invalid" which is maybe not good either. I'm not sure about the security impact of clearly state which of the username or the password is wrong, but I agree with you that it's annoying from a user perspective. I'll ask some friends who work in security for their insight.

Reply:
Considering that an attacker can test the existence of a username by trying to create an account, I would say there are no reasons to hide the exact information

Reply:
That's a great point. :+1: